provider/lxd/credentials.go

```diff
@@ -6,18 +6,48 @@
 package lxd
 
 import (
+	"net"
+
+	"github.com/juju/errors"
+
 	"github.com/juju/juju/cloud"
 	"github.com/juju/juju/environs"
+	"github.com/juju/juju/tools/lxdclient"
 )
 
-type environProviderCredentials struct{}
+const (
+	credAttrServerCert = "server-cert"
+	credAttrClientCert = "client-cert"
+	credAttrClientKey  = "client-key"
+)
+
+type environProviderCredentials struct {
```

Comment:
given that the purpose of this struct is not 100% obvious at first read, could we add a comment, even if just to say that it implements the ProviderCredentials interface